src/index.ts

```diff
@@ -180,8 +180,21 @@ export async function install(
   return true;
 }
 
+/**
+ * Returns CLI arguments configured in the package.json
+ */
+function packageCfg() {
+  try {
+    return require(path.join(cwd, 'package.json'))['@pika/web'] || {};
+  } catch (e) {
+    return {};
+  }
+}
+
 export async function cli(args: string[]) {
-  const {help, optimize = false, strict = false, clean = false, dest = 'web_modules'} = yargs(args);
+  const {help, optimize = false, strict = false, clean = false, dest = 'web_modules'} = 
+    // overwrite package.json arguments with CLI arguments
+    Object.assign(packageCfg(), yargs(args));
```

Comment:
so we actually load the package.json manifest right below this into `cwdManifest`. Can you move that line up so that it happens before we destructure the args?

```ts
const cwdManifest = require(path.join(cwd, 'package.json'));
const {help, webDependencies, optimize = false, strict = false, clean = false, dest = 'web_modules'} = {
  ...yargs(args),
  ...(cwdManifest['@pika/web'] || {})
};
```